bin/rad-issue

```diff
@@ -22,22 +22,31 @@
 
 (def help
   (string-append
-  "rad-issue - Radicle issue CLI
+  "rad issue - Radicle Issue CLI
 
    Usage:
-        rad-issue list " list-opts-help "
-        rad-issue new
-        rad-issue [show | close] <issue-number>
-        rad-issue comment <issue-number> <comment>
-        rad-issue help
-
-     list         - Select from issues to view or comment on
+        rad issue list " list-opts-help "
+        rad issue new
+        rad issue [show | close] <issue-number>
+        rad issue comment <issue-number> <comment>
+        rad issue help
+
+     list         - Lists all issues
+                    The list can be filtered by state:
```

Comment:
Maybe extract this string to a function to avoid duplication with the rad-diff help.

Reply:
I would leave it for now. Maybe we could have a future PR where we also extract all non-error user feedback to make them more consistent.